src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,171 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    private static final int ARRAY_YEAR = 0;
```

Comment:
```suggestion
    private static final int YEAR_INDEX = 0;
```
to me, this seems more understandable

Reply:
You are absolutely right, done